index.d.ts

```diff
@@ -1 +1,6 @@
-export default class ManyKeysMap<K, V> extends Map<K[], V> {}
+export default class ManyKeysMap<K extends readonly unknown[], V> extends Map<K, V> {
```

Comment:
@mshriner I think this is the right way to enforce tuples here.

I'm not publishing this yet, I can accept a PR with further tests now that the whole testing suite is in TS